src/utils/config.cpp

```diff
@@ -120,7 +121,41 @@ void Config::ParseEmbrakes(char* line)
   }
 }
 
-void Config::ParseSensors(char* line)
+// if there is no creator configured to an interface, we use this one to prevent calling
+// a null pointer function
+template<class T>
+T* createDefault()
+{
+  printf("ERROOOR: no creator for %s found, creating NULL object\n", interfaceName<T>());
+  return NULL;
+}
+
+void Config::parseFactory(char* line)
+{
+  // parse into key value pair, validate input line
+  char* key   = strtok(line, " ");
+  char* value = strtok(NULL, " ");
```

Comment:
Maybe add a comment explaining this continues parsing the previous string passed to it? I had to look it up in the docs and I still find it a rather strange.
```suggestion
  char* value = strtok(NULL, " ");  // `NULL` means `strtok` continues parsing the last string passed to it (`line`)
```

Reply:
See comment at the beginning of `parseSensors()` function. This has been already described. Do you think we shall describe it again at every point of usage?

Reply:
Ah, ok. No, I would not duplicate that. I suppose sensors is the best example but it's a pity that it comes last.